docs/products/m3db/howto/telegraf.rst

```diff
@@ -0,0 +1,51 @@
+How to Write to M3 from Telegraf
+================================
+
+M3 supports multiple methods of writing metrics to the M3 database. Two common options are the InfluxDB wire-protocol, and the Prometheus remote write protocol. It is possible to write metrics from an existing Telegraf setup to M3 using either of these protocols, so we've covered the configuration for both options here.
+
+Configuring Telegraf InfluxDB output plugin for M3
+--------------------------------------------------
+
+This is how you configure Telegraf to send metrics to M3 using the InfluxDB line-protocol. This configuration needs to be part of the output plugins configuration in your Telegraf configuration file::
```

Comment:
"This **configuration** needs to be part of the output plugins **configuration** in your Telegraf **configuration** file::" 
->
"The following lines need to be part of the output plugins section in your Telegraf configuration file::"

Reply:
Ha, that is a bit difficult to digest! Updated.